src/registrar/context_processors.py

```diff
@@ -70,16 +66,22 @@ def portfolio_permissions(request):
                 "has_base_portfolio_permission": False,
                 "has_domains_portfolio_permission": False,
                 "has_domain_requests_portfolio_permission": False,
+                "portfolio": None,
+                "has_organization_feature_flag": False,
             }
         return {
             "has_base_portfolio_permission": request.user.has_base_portfolio_permission(),
             "has_domains_portfolio_permission": request.user.has_domains_portfolio_permission(),
             "has_domain_requests_portfolio_permission": request.user.has_domain_requests_portfolio_permission(),
+            "portfolio": request.user.portfolio,
+            "has_organization_feature_flag": flag_is_active(request, "organization_feature"),
         }
```

Comment:
I don't know.  I think this is a good thought.  I was honestly hesitant to use global context for anything that hits the database because the load it would put on every request.  But decided to do this since these variables need to be available in context for most user facing views.